e2e/test-basic.ts

```diff
@@ -0,0 +1,17 @@
+import { assert } from 'chai'
```

Comment:
Could you put this file in test/e2e ? I prefer all test-related stuff to be under "test".